packages/ndla-ui/src/Embed/conceptComponents.tsx

```diff
@@ -182,20 +184,21 @@ export const ConceptNotionV2 = forwardRef<HTMLDivElement, ConceptNotionProps>(
     ref,
   ) => {
     const { t } = useTranslation();
-
+    const isGloss = conceptType === 'gloss';
     return (
       <div css={inPopover ? notionContentCss : undefined} {...rest} ref={ref}>
         <ContentPadding>
-          <NotionHeader>
+          <NotionHeader data-show-separator={!isGloss}>
             <h1>
-              {title.title} {<small>{t(`searchPage.resultType.${conceptType}`)}</small>}
+              {!isGloss && title.title}
+              {<small data-show-separator={!isGloss}>{t(`searchPage.resultType.${conceptType}`)}</small>}
```

Comment:
Det at dette endres fra ei til to linjer, gjør det at vi mister mellomrommet mellom tittel og small?

Reply:
Mellomrommet legges på fra css